rust/kcl-lib/tests/kcl_samples/cold-plate/program_memory.snap

```diff
@@ -1651,112 +1651,117 @@ description: Variables in memory after executing cold-plate.kcl
     }
   },
   "tubeWall": {
-    "type": "Solid",
-    "value": {
-      "type": "Solid",
-      "id": "[uuid]",
-      "artifactId": "[uuid]",
-      "value": [
-        {
-          "faceId": "[uuid]",
+    "type": "HomArray",
+    "value": [
+      {
+        "type": "Solid",
+        "value": {
+          "type": "Solid",
           "id": "[uuid]",
-          "sourceRange": [],
-          "tag": null,
-          "type": "extrudeArc"
-        }
-      ],
-      "sketch": {
-        "type": "Sketch",
-        "id": "[uuid]",
-        "paths": [
-          {
-            "__geoMeta": {
+          "artifactId": "[uuid]",
+          "value": [
+            {
+              "faceId": "[uuid]",
               "id": "[uuid]",
-              "sourceRange": []
-            },
-            "ccw": true,
-            "center": [
-              -3.0,
-              0.625
-            ],
-            "from": [
-              -2.688,
-              0.625
-            ],
-            "radius": 0.3125,
-            "tag": null,
-            "to": [
-              -2.688,
-              0.625
+              "sourceRange": [],
+              "tag": null,
+              "type": "extrudeArc"
+            }
+          ],
+          "sketch": {
+            "type": "Sketch",
+            "id": "[uuid]",
+            "paths": [
+              {
+                "__geoMeta": {
+                  "id": "[uuid]",
+                  "sourceRange": []
+                },
+                "ccw": true,
+                "center": [
+                  -3.0,
+                  0.625
+                ],
+                "from": [
+                  -2.688,
+                  0.625
+                ],
+                "radius": 0.3125,
+                "tag": null,
+                "to": [
+                  -2.688,
+                  0.625
+                ],
+                "type": "Circle",
+                "units": {
+                  "type": "Inches"
+                }
+              }
             ],
-            "type": "Circle",
+            "on": {
+              "artifactId": "[uuid]",
+              "id": "[uuid]",
+              "origin": {
+                "x": -186.68999999999997,
```

Comment:
Should we add a redaction rule to round this?

Reply:
yes :-)